marquez_client/log_backend.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from marquez_client.backend import Backend
+from marquez_client.utils import Utils
+
+log = logging.getLogger(__name__)
+
+
+class LogBackend(Backend):
+    def __init__(self):
+        log.debug("LogBackend.init")
+
+    def put(self, path, headers, json):
+        log.debug("_put()")
+
+        put_details = {}
+
+        put_details['method'] = 'PUT'
+        put_details['path'] = path
+        put_details['headers'] = headers
+        put_details['payload'] = json
+
+        Utils.log_details(put_details)
+
+    def post(self, path, headers, json=None):
+        log.debug("_post()")
+
+        post_details = {}
+
+        post_details['method'] = 'POST'
+        post_details['path'] = path
+        post_details['headers'] = headers
+        post_details['payload'] = json
```

Comment:
Should have a check for `json=None` and not set `payload` if that's the case?

Reply:
it breaks previous contacts. is it okay ?

Reply:
I just wondering what the default payload should be. Rather than, `None` we'd probably want an empty json object `{}`

Reply:
done.